src/pages/ProAmmPool/PositionListItem.tsx

```diff
@@ -270,8 +304,7 @@ function PositionListItem({ stakedLayout, farmAvailable, positionDetails, refe }
                 position={position}
                 tokenId={positionDetails.tokenId}
                 layout={1}
-                farmAvailable={farmAvailable}
-                disableCollectFee={!!stakedLiquidity}
```

Comment:
`disableCollectFee` is not used anymore, isnt it?

Reply:
Yeah, i removed it